i18n/en.json

```diff
@@ -3048,6 +3049,10 @@
   "create_category_modal.createCategory": "Create New Category",
   "create_comment.addComment": "Reply to this thread...",
   "create_comment.commentTitle": "Comment",
+  "create_group_memberships_modal.cancel": "No",
+  "create_group_memberships_modal.create": "Yes",
+  "create_group_memberships_modal.desc": "This will add (or re-adds) {username} to all of their teams and channels that they get added to based on their group memberships. Do you want to proceed?",
```

Comment:
```suggestion
  "create_group_memberships_modal.desc": "This will add (or re-add) {username} to all of the available teams and channels based on their group memberships. You can revert this change at any time.",
```